rusqlite_migration/src/lib.rs

```diff
@@ -419,7 +421,8 @@ impl<'m> Migrations<'m> {
         let migrations = from_directory(dir)?
             .into_iter()
             .collect::<Option<Vec<_>>>()
-            .ok_or(Error::FileLoad("Could not load migrations".to_string()))?;
+            .ok_or(Error::FileLoad("Could not load migrations".to_string()))?
+            .into();
```

Comment:
```suggestion
        let migrations = from_directory(dir)?
            .into_iter()
            .collect::<Option<Cow<_>>>()
            .ok_or(Error::FileLoad("Could not load migrations".to_string()))?;
```
nit: we could avoid the intermediary Vec.